awx/main/conf.py

```diff
@@ -54,15 +54,6 @@
     category_slug='system',
 )
 
-register(
-    'TOWER_ADMIN_ALERTS',
-    field_class=fields.BooleanField,
-    label=_('Enable Administrator Alerts'),
-    help_text=_('Email Admin users for system events that may require attention.'),
-    category=_('System'),
-    category_slug='system',
-)
```

Comment:
Have we ever tested removing a setting? This is where I get worried about how the UI would PATCH with the default values, so it's almost certain the pre-migration this value will be set for anyone who ever saved settings. Should be simple to test.

Reply:
Yep, removing settings is fine as long (as you don't reference them in code somewhere).  Having stray records in the `conf_setting` table shouldn't hurt anything.